.github/workflows/ci-rubocop.yml

```diff
@@ -0,0 +1,54 @@
+name: "CI - Rubocop"
+
+on:
+  push:
+    paths:
+      # NOTE: GitHub Actions do not allow using YAML references, the same path
+      # list is used below for the pull request event. Keep both lists in sync!!
+
+      # this file as well
+      - .github/workflows/ci-rubocop.yml
+      # Rubocop configuration
+      - service/.rubocop.yml
+      # all Ruby files
+      - service/Gemfile
+      - service/bin/agamactl
```

Comment:
@teclator is about to add `service/bin/agama-setup-proxy` file, so perhaps we should use `bin/*`.